dev/PushNotifications/PushNotificationTelemetry.h

```diff
@@ -31,8 +32,9 @@ class PushNotificationTelemetry : public wil::TraceLoggingProvider
                 _GENERIC_PARTB_FIELDS_ENABLED,
                 TraceLoggingHexUInt32(hr, "OperationResult"),
                 TraceLoggingWideString(to_hstring(remoteId).data(), "RemoteId"),
```

Comment:
Nit: Is TraceLoggingGuid supported in WAS?

Reply:
done